.drone.yml

```diff
@@ -598,13 +598,17 @@ steps:
   - cd $WORKSPACE_DIR/go/src/github.com/gravitational/webapps
   - yarn install && yarn build-term && yarn package-term -c.extraMetadata.version=$VERSION
   environment:
+    APPLE_PASSWORD:
```

Comment:
From memory, the darwin build process automatically signs things if these `APPLE_*` environment values are set. Is that the core of the change for producing signed binaries?